db/migrate/20160218001400_add_default_follow_up_layout_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultFollowUpLayoutId < ActiveRecord::Migration
+  def change
+    add_reference :liquid_layouts, :default_follow_up_layout
```

Comment:
the index part was absolutely just a sincere question and your answer makes sense to me. The other part is that I'm surprised that rails can figure out that `:default_follow_up_layout` is a reference to a row in the same table without telling it. I think whats happening is that postgres has no idea it's a foreign key, the association just happens entirely in ruby